src/bin/index.ts

```diff
@@ -90,6 +91,7 @@ function parseCliArgs(argv: string[]) {
     sourcemap: !!args['--sourcemap'],
     cwd: args['--cwd'],
     dts: args['--no-dts'] ? false : undefined,
+    dtsBundle: args['--dts-bundle'],
```

Comment:
Instead of introducing a new option, we can use `--no-external` to achieve it.
If we bundle all the dependencies, then we bundle all the types to align as well.

Reply:
Per `rollup-plugin-dts`, enabling bundle dts is not recommended. So IMHO we should split this into a separate option.

Reply:
yea but I still believe users want to have types as bundled instead of relying on separate package when they specify `--no-external`.

I want to support types bundling for that case too. if types not working they can have a way to opt-out types bundling. I'll leave these items for later. Current approach looks good! Thanks